### PR TITLE
New App: SamuelElliott.NintendoSwitchOnline version 1.6.1

### LINKED ISSUE #103727: [Package Request]: nxapi (Nintendo Switch Discord presence)
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/samuelthomas2774/nxapi
Publisher: samuelthomas2774
Package Name: nxapi
Description: JavaScript library and command line and Electron app for accessing the Nintendo Switch Online and Nintendo Switch Parental Controls app APIs. Show your Nintendo Switch presence in Discord, get friend notifications on desktop, and download and access SplatNet 2, NookLink, SplatNet 3 and Parental Controls data.
Package Version: 1.6.1
Installer URL: https://github.com/samuelthomas2774/nxapi/releases/download/v1.6.1/Nintendo.Switch.Online.Setup.1.6.1.exe

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves #103727 

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103806)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Diagnosing the pipelines.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie, it look like something weird on this pipelines. 

Screenshot (214)

**Comment by BrandonWanHuanSheng** [comment]
Most likely, this one need pinning.

**Comment by stephengillie** [comment]
`x86` Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230428161450

File: manifests/s/SamuelElliott/NintendoSwitchOnline/1.6.1/SamuelElliott.NintendoSwitchOnline.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /S
+  Custom: /CurrentUser
+AppsAndFeaturesEntries:
+- DisplayName: Nintendo Switch Online 1.6.1
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/s/SamuelElliott/NintendoSwitchOnline/1.6.1/SamuelElliott.NintendoSwitchOnline.installer.yaml:18]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**